nautilus: msg/simple: reset in_seq_acked to zero when session is reset

penglaiyxy opened the pull request:
SimpleMessenger module  has been removed from master branch in e57af1d, so I picked the nautilus branch to merge in.

Fixes:  https://tracker.ceph.com/issues/41195
Signed-off-by:  Xiangyang Yu penglaiyxy@gmail.com

penglaiyxy commented:
@tchaikov  The label is  added "mgr"? I think it should be "msg".

tchaikov commented:
thanks @penglaiyxy! fixed.

tchaikov commented:
@penglaiyxy the fix looks great! a couple things though:

1. replace "[msg/simple]" with "msg/simple" in the title of the commit message
1. add a section in the commit message explaining why this change is not cherry-picked from master -- i think it's because that master has removed SimpleMessenger in e57af1d4
1. target this change to mimic
1. could you file a ticket on tracker so we can backport it to nautilus
1. and add a "Fixes:" line in the commit message to link the commit to that tracker ticket.

penglaiyxy commented:
@tchaikov  I have modified my commit title and added a section explaining why I picked the nautilus branch to merge in .

Ticket in tracker has been added too.

Can we backport it from nautilus to mimic ?

tchaikov commented:
@penglaiyxy could you remove "nautilus:" from the title of your commit message?

> Can we backport it from nautilus to mimic ?

sorry i was wrong. thought you were doing the other way around. =(

so all good =)

smithfarm commented:
@penglaiyxy As @tchaikov said - please change the commit title to:

```
msg/simple: reset in_seq_acked to zero when session is reset
```

(note the space after `msg/simple: `)

penglaiyxy commented:
@smithfarm @tchaikov  Commit message has been modified.

yuriw commented:
![](https://github.trello.services/images/mini-trello-icon.png) [wip-yuri5-testing-2019-11-07-2203-nautilus](https://trello.com/c/74sn1TLH/751-wip-yuri5-testing-2019-11-07-2203-nautilus)

yuriw commented:
![](https://github.trello.services/images/mini-trello-icon.png) [wip-yuri4-testing-2019-11-nautilus](https://trello.com/c/HMv8KY2i/755-wip-yuri4-testing-2019-11-nautilus)
